java/cleanup.sh

```diff
@@ -1,10 +1,6 @@
-pkill -9 python
-pkill -9 local_scheduler
-pkill -9 plasma_manager
-pkill -9 plasma_store_server
-pkill -9 global_scheduler
-pkill -9 redis-server
-pkill -9 redis
-pkill -9 raylet
-ps aux | grep ray | awk '{system("kill "$2);}'
-rm /tmp/raylet*
+# Stop backend processes
+ray stop
```

Comment:
Could this command `ray stop`  be found if we didn't install ray-python version?

Reply:
I suppose not. One option would be to rewrite the `ray stop` script as a shell script, which can be called here. However, that may make it hard to improve the script (e.g., to make sure we're just killing the right processes (see https://github.com/ray-project/ray/issues/2830).

We often have to convert shell scripts to Python eventually.

Reply:
I agree it's preferable to rewrite the script with shell script. But using `ray stop` should be okay for now as a quick fix. If we do need to install Java alone without Python, I can change this in the future.